include/clap/universal-plugin-id.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+// Pair of plugin ABI and plugin identifier.
+//
+// If you want to represent other formats please send us an update to the comment with the
+// name of the abi and the representation of the id.
+typedef struct clap_universal_plugin_id {
+   // The plugin ABI name, in lowercase.
+   // eg: "clap", "vst3", "vst2", "au", ...
```

Comment:
should we mention "null-terminated" here?

Reply:
Yes, that'd be better.

Reply:
see #387